File: java/src/test/java/org/ikigaidigital/TimeDepositCalculatorTest.java
package org.ikigaidigital;

import org.junit.jupiter.api.Test;

import java.util.*;

import static org.assertj.core.api.Assertions.assertThat;

public class TimeDepositCalculatorTest {
    @Test
    public void updateBalance_Test() {
        TimeDepositCalculator calc = new TimeDepositCalculator();
        List<TimeDeposit> plans = Arrays.asList(
            new TimeDeposit(1,"basic", 1234567.00, 45)
        );
        calc.updateBalance(plans);

        assertThat(1).isEqualTo(1);
    }
}

File: kotlin/src/test/kotlin/org/ikigaidigital/TimeDepositCalculatorTest.kt
package org.ikigaidigital

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class TimeDepositCalculatorTest {
    @Test
    fun updateBalance_Test() {
        val calc = TimeDepositCalculator()
        val plans = listOf(
            TimeDeposit(1, "basic", 1234567.00, 45)
        )
        calc.updateBalance(plans)
        Assertions.assertThat(1).isEqualTo(1)
    }
}